packages/pay/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "@interledger/pay",
+  "description": "Send payments over Interledger",
+  "version": "0.1.0",
+  "author": "Interledger Team <info@interledger.org>",
+  "license": "Apache-2.0",
+  "publishConfig": {
+    "access": "public"
+  },
+  "main": "dist/src/index.js",
+  "types": "dist/src/index.d.ts",
+  "files": [
+    "dist/**/*.js",
+    "dist/**/*.js.map",
+    "dist/**/*.d.ts",
+    "!dist/test/**/*"
+  ],
+  "scripts": {
+    "build": "tsc -p tsconfig.json",
+    "test": "jest",
+    "cover": "jest --coverage --ci",
+    "codecov": "codecov --root=../../ -f coverage/*.json -F pay"
+  },
+  "dependencies": {
+    "@rgrove/parse-xml": "^2.0.2",
+    "axios": "^0.19.2",
+    "bignumber.js": "^9.0.0",
+    "ilp-logger": "^1.3.0-alpha.1",
+    "ilp-packet": "^3.1.0-alpha.0",
+    "ilp-protocol-ildcp": "^2.1.4",
+    "ilp-protocol-stream": "^2.3.1",
+    "long": "^4.0.0",
+    "uuid": "^7.0.2"
+  },
+  "devDependencies": {
+    "@kincaidoneil/ilp-connector": "^23.2.0",
```

Comment:
I'll submit a PR to `ilp-connector` with my changes. Since `ilp-connector` bundles the source TS, it caused build errors, I had to change it to only publish the transpiled JS and types.